tt_metal/impl/dispatch/kernels/cq_prefetch.cpp

```diff
@@ -1086,9 +1089,11 @@ template<bool cmddat_wrap_enable,
          bool exec_buf>
 bool process_cmd(uint32_t& cmd_ptr,
                  uint32_t& downstream_data_ptr,
-                 uint32_t& stride) {
-
-    DeviceZoneScopedND("PROCESS-CMD", block_noc_writes_to_clear, rd_block_idx );
+                 uint32_t& stride,
+                 uint32_t* l1_cache,
+                 PrefetchExecBufState& exec_buf_state) {
+    RECORD_STACK_USAGE();
```

Comment:
Explicit call here is to check prefetch stack usage outside of just when the kernel finishes? Do we need to do the same for dispatch?

Reply:
Kept this in the PR by mistake. This was meant as a temporary change to verify that we weren't overflowing the stack. Thanks for pointing it out.